import_export_extensions/models/export_job.py

```diff
@@ -159,13 +159,6 @@ def export_filename(self) -> str:
     def progress(self) -> TaskStateInfo | None:
         """Return dict with parsing state.
 
-        Example for sync mode::
```

Comment:
Let's also update part about "celery (celery) mode". I guess now it's relevant for any mode, so just remove `celery` mentions.

Also I see "Possible states" in docstring, but can we also add this enum to `TaskStateInfo`?